app/scripts/components/common/mapbox/layers/raster-timeseries.tsx

```diff
@@ -63,7 +63,7 @@ export function MapLayerRasterTimeseries(props: MapLayerRasterTimeseriesProps) {
 
   const theme = useTheme();
   const primaryColor = theme.color?.primary;
-  const minZoom = zoomExtent?.[0];
+  const minZoom = zoomExtent?.[0] ?? -Infinity;
```

Comment:
Careful here - `0` would work just as well and this could cause trouble on line 219.

Reply:
You're absolutely right! Thank you :)